deploy/role.yaml

```diff
@@ -11,14 +11,47 @@ rules:
   - ""
   resources:
   - pods
-  - services
-  - services/finalizers
-  - endpoints
+  verbs:
+  - get
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
```

Comment:
why need to create configmap?

Reply:
operator will use it to create a lock for leader selection.

Reply:
sure
sorry